### ISSUE TITLE
debezium-db2

### ISSUE THREAD

**Issue by first-better** [naridnevahgar/cdc-debezium-db2-mysql#1]
Hello!
Could you tell me the version of db2,kafka,debezium that you used in cdc ?

I can connect db2 with kafka by use  db2(11.5),kafka(2.4.1). But I just get the snapshot, I cannot  get data when I insert or update data in db2 table. I am very confused that  log and status are all OK.

**Comment by naridnevahgar** [comment]
Hi @first-better, i used debezium v1.0. When i tried this, the DB2 connector was not GA. Did you check the kafka topics to see if the new topics are created against table names?

**Comment by first-better** [comment]
Hi,  I checked. The topic has been created. When I consume the data in the topic, I just get the data that was in the table when I commit the job.  But  I cannot get data when I insert or update data in db2 table.  By the way,did you hive IBM InfoSphere Data Replication (IIDR)? Is it necessary?

**Comment by naridnevahgar** [comment]
In my case, I used the docker version of db2, the official image from IBM. So, I used the utility provided by Debezium team to put the tables under capture. 

Based on your explanation, what i observe is the initial snapshot works perfectly. However, the further CDC captures don't emit events. Under the hood, the connector relies on the DB2 transaction logs and converts them as events. So confirm if your tables are still under capture mode and check if your updates create the transaction logs as expected.

**Comment by first-better** [comment]
OK, I will check it. 
Thank you!

**Comment by first-better** [comment]
Hi!
I got this WARN:
WARN No maximum LSN recorded in the database; please ensure that the DB2 Agent is running (io.debezium.connector.db2.Db2StreamingChangeEventSource:122)
I am sure that db2 is running. 
And, did you test this by using your own virtual machine?

**Comment by first-better** [comment]
HI, 
did you change some configuration in your db2 database?
I guess the problem I'm having is probably caused by the database

**Comment by naridnevahgar** [comment]
Ok, Now it is very clear that the DB2 instance is not capturing the transaction logs against your source tables. 
The ASN process is responsible to create the logs, which hold the sequence number (LSN).

In my case, since I used the docker image on my local, I used [the tool from Debezium](https://debezium.io/documentation/reference/connectors/db2.html#setting-up-db2). The cli tool has commands to put tables under capture mode. This is a prerequisite for the setup.
